newIDE/app/src/UI/SortableVirtualizedItemList/index.js

```diff
@@ -42,20 +42,53 @@ export default class SortableVirtualizedItemList<Item> extends React.Component<
     if (this._list) this._list.forceUpdateGrid();
   }
 
+  renderItemRow(
```

Comment:
Nitpicking: `_renderItemRow`, to emphasise that it's not to be called from the outside.